lib/config.js

```diff
@@ -616,8 +631,8 @@ const tryToLoad = (configPath, fallbackHandler) => {
     if (!config) {
       return Promise.reject(new Error(`${basename} must be a valid object`));
     }
-    if (!config.files) {
-      return Promise.reject(new Error(`${basename} must have "files" property`));
+    if (!('files' in config)) {
+      return Promise.reject(new Error(`${basename} must have "files" property. Here's a minimal brunch-config.js to get you started:\n${minimalConfig}`));
```

Comment:
Above, `"package.json"` was written with quotes, but here `brunch-config.js` (and possibly `${basename}`?) is written without quotes. I know, this is really nitpicky, but perhaps it's cleanest to consistently drop the quotes?